docs/UserGuide.md

```diff
@@ -310,59 +324,96 @@ Example command usages from the current student list shown in Figure 1:
-Use <code>find_student</code> command to filter the list of students you want.<br>
-The <code>STUDENT_INDEX</code> will be based on the current list.<br>
+Similar to <code>edit_student</code>, you can also use <code>find_student</code> command to filter the list of students you want.<br>
+The <code>STUDENT_INDEX</code> will be based on the current / filtered list.<br>
 (E.g. If John Doe is the 500th student, instead of <code>delete_student 500</code>, you can use <code>find_student John Doe</code> and
 <code>delete_student 1</code> if the John Doe you would like to delete is the 1st in the list)
 </div>
 
-Examples:
-* `delete_student 2` deletes the 2nd student based on the current list of students
-
 ##### Listing students' emails based on current list: `emails`
 As a user, you might want to quickly get all the email address of your students and send them a mass email.
-This feature displays a concatenated string of students' emails based on current list, separated by `;` which can be copied to their respective email client (E.g. gmail) for mass sending of emails to students.
+This feature displays a concatenated text of students' emails based on current list, separated by `;` which can be copied to their respective email client (E.g. Gmail) for mass sending of emails to students.
 
 Format: `emails`
 
 Example:
 
-\# | Student Name | Email
----- |---------|------|
-1 | John Lee | johnlee@gmail.com
-2 | Johnz Tan | johnztan@gmail.com
-3 | Jon Koh | jonkoh@gmail.com
-4 | Samuel Lee | sam@gmail.com
+1. You want to send a mass email to **all** of your students. 
+
+2. Type `emails` and press <kbd>enter</kbd>.
 
-*Figure 2.1 State of Student List After `list` command*
+![emails_example](images/enhao/email%20-%20edited.png)
 
-\# | Student Name | Email
----- |---------|------|
-1 | John Lee | johnlee@gmail.com
-2 | Jon Koh | jonkoh@gmail.com
+3. The concatenated text of all the students' emails will be displayed.
 
-*Figure 2.2 State of Student List After `find_student john jon` command*
+![emails_result_example](images/enhao/email_result%20-%20edited.png)
 
-* To get emails of all students (see Figure 2.1): `list` followed by `emails` returns `johnlee@gmail.com;johnztan@gmail.com;jonkoh@gmail.com;sam@gmail.com;`
+4. The user can copy the values in the result display box to the "To" field of any email client that they have (E.g. Gmail).
+
+![emails_client_example](images/enhao/email_client-edited.png)
+
+<div markdown="span" class="alert alert-primary">:bulb: Tip:
+Similar to <code>edit_student</code>, you can also use <code>find_student</code> command after step 1 to filter the list of students you want.<br>
+The email addresses displayed will be based on the current list.<br>
+(E.g. If you only want the the email addresses of <code>Alex Yeoh</code>, <code>David Li</code> and <code>Roy Balakrishnan</code> from the sample data,
+, you can use <code>find_student Alex David Roy</code> and <code>emails</code>
```

Comment:
there's an extra comma here

Reply:
Nice catch! Updated in the latest commit.